### ISSUE TITLE
Test set 

### ISSUE THREAD

**Issue by Swati17293** [AIRobotZhang/STCKA#6]
Whenever we try to use the saved model, the results will be different as each time the training data is randomly split. In addition, the arguments provided for the test and the dev set are not used. When we use train-test-split from sklearn, we can set "random state = 0" but since you use your own function for the same, you can suggest how to address the issue.
Thank you for sharing the code!!!!!!!

**Comment by AIRobotZhang** [comment]
> Whenever we try to use the saved model, the results will be different as each time the training data is randomly split. In addition, the arguments provided for the test and the dev set are not used. When we use train-test-split from sklearn, we can set "random state = 0" but since you use your own function for the same, you can suggest how to address the issue.
> Thank you for sharing the code!!!!!!!

Sorry for having confused you. I think splitting the all data into train and test data in advance can be an idea, then only using the dataset.train_dev_split function for the train data can be ok. Please feel free to let me know if you have any other questions.
